### ISSUE TITLE
Approach to select appropriate priors for generation and sampling time distributions

### ISSUE THREAD

**Issue by dpenas-u** [xavierdidelot/TransPhylo#30]
Hi Didelot,

I am using the infer_multittree function in the TransPhylo package in R to infer transmission in a group of Mycobacterium tuberculosis clusters from a specific region. However, I am not sure which distribution to use for the generation and sampling time distributions for all the clusters. To address this, I plan to run the inference individually for each cluster, varying the prior distributions for the generation and sampling time distributions. Based on the literature, I have observed that the generation and sampling time distribution ranges from 1 to 2.5. Therefore, I will try different shape values within that range and distributions with different tails (scale parameter). I will examine how well the posterior distributions fit with the prior distributions and use this information to obtain the best values for the generation and sampling time parameters (with higher fitting) for all the clusters.

My question is whether this approach is reasonable for estimating the generation and sampling time parameters. Once I have obtained the best values for all clusters, I plan to perform multi-inference with those values.

Thank you very much in advance for your help, and I apologize for any inconvenience.

**Comment by sriram98v** [comment]
The mean of the gamma distribution presenting in your paper (TB experiment) comes to 0.39 (1.3 * 0.3). Does this equate to saying that the average generation time of TB is 0.39 years?

**Comment by xavierdidelot** [comment]
The paper states that we "used a Gamma distribution with shape parameter 1.3 and rate parameter 0.3"
In other words this has shape 1.3 and scale 1/0.3=3.333
The mean is shape * scale=1.3*3.333=4.33 years
The 95% range is from 0.23 to 14.31 years

**Comment by sriram98v** [comment]
If I am working with a pathogen with a smaller generation time (say 2 months), should I select the parameters of the gamma distribution such that the mean is ~0.16 (2/12)?

**Comment by xavierdidelot** [comment]
Yes that's right, and if you also consider the variance you want around this mean then you can deduce both scale and shape parameters as explained previously in this thread.

**Comment by sriram98v** [comment]
Understood. I appreciate your help!

**Comment by liamxg** [comment]
Dear @xavierdidelot ,

Could I know what is the rate parameter mean here: "used a Gamma distribution with shape parameter 1.3 and rate parameter 0.3", thanks.

Best,
Liam

**Comment by xavierdidelot** [comment]
Maybe you are more used to the parametrisation of the Gamma distribution using shape and scale. The rate is equal to 1/scale.

**Comment by liamxg** [comment]
how to calculate the shape and scale if I only know the mean?

**Comment by xavierdidelot** [comment]
You can't. 
But if you know the mean and the variance (or standard deviation) then you can.

**Comment by liamxg** [comment]
then we do not need to calculate the shape and scale, use w.mean and w.std, right?

**Comment by xavierdidelot** [comment]
Yes indeed you can use that and it is equivalent
